AUTHORS.rst

```diff
@@ -4,6 +4,7 @@
 - Brendan Smithyman, (`@bsmithyman <https://github.com/bsmithyman/>`_)
 - Gudni Rosenkjaer, (`@grosenkj <https://github.com/grosenkj/>`_)
 - Dom Fournier, (`@fourndo <https://github.com/fourndo/>`_)
+- Joseph Capriotti, (`@jcapriot <https://github.com/jcapriot`)
```

Comment:
🎉

Reply:
I think the final `` ` `` should be a ``>`_`` in your name and John's name

Reply:
yup...